app-modules/student-data-model/resources/views/filament/resources/educatable-resource/widgets/educatable-activity-feed-widget.blade.php

```diff
@@ -31,90 +31,31 @@
-                            </p>
-                        </div>
-
-                        @if ($user = $this->getTimelineRecordUser($record->timelineable))
-                            <div class="mt-0.5 shrink-0">
-                                <x-filament::badge
-                                    color="gray"
-                                    size="sm"
-                                >
-                                    {{ $user->name }}
-                                </x-filament::badge>
-                            </div>
-                        @endif
-                    </div>
-
-                    <p class="truncate text-sm text-gray-500 dark:text-gray-400">
-                        {{ $this->getTimelineRecordDescription($record->timelineable) }}
-                    </p>
-                </div>
-            </button>
-        @empty
-            <div class="p-6">
-                <div class="mx-auto grid max-w-lg justify-items-center gap-4 text-center">
-                    <div class="rounded-full bg-gray-100 p-3 dark:bg-gray-500/20">
-                        @svg('heroicon-o-inbox-arrow-down', 'h-6 w-6 text-gray-500 dark:text-gray-400')
-                    </div>
-
-                    <h4 class="text-base font-semibold leading-6 text-gray-950 dark:text-white">
-                        No activities
-                    </h4>
-                </div>
+<section
+    class="fi-section rounded-xl bg-white shadow-sm ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10"
+    data-has-alpine-state="true"
+    x-data="{
+        isCollapsed: false,
+    }"
+    wire:poll.10s=""
+>
+    <header class="fi-section-header flex flex-col gap-3 px-6 py-4">
+        <div class="flex items-center gap-3">
+            <div class="grid flex-1 gap-y-1">
+                <h3 class="fi-section-header-heading text-base font-semibold leading-6 text-gray-950 dark:text-white">
+                    Activity Feed
+                </h3>
             </div>
-        @endforelse
-    </x-filament::section>
-</x-filament-widgets::widget>
+            <a
+                class="fi-btn fi-btn-color-gray fi-color-gray fi-size-md fi-btn-size-md [input:checked+&amp;]:bg-gray-400 [input:checked+&amp;]:text-white [input:checked+&amp;]:ring-0 [input:checked+&amp;]:hover:bg-gray-300 dark:[input:checked+&amp;]:bg-gray-600 dark:[input:checked+&amp;]:hover:bg-gray-500 relative inline-grid grid-flow-col items-center justify-center gap-1.5 rounded-lg bg-white px-3 py-2 text-sm font-semibold text-gray-950 shadow-sm outline-none ring-1 ring-gray-950/10 transition duration-75 hover:bg-gray-50 focus-visible:ring-2 dark:bg-white/5 dark:text-white dark:ring-white/20 dark:hover:bg-white/10"
+                href={{ $viewUrl }}
+                style=";"
+            >
+                <span class="fi-btn-label">
+                    Full Feed
+                </span>
+            </a>
```

Comment:
This should be converted to use the `<x-filament::button>` Blade component, as before. Make sure it only renders `@if ($timelineRecords)`